AngularPart1/Components/Home/home.html

```diff
@@ -0,0 +1,26 @@
+<div ng-controller="toDoController">
+    <h3>Active ({{ tasks.length }})</h3>
+    <span ng-if="tasks.length === 0">You have no active tasks!</span>
+    <div class="filters">
+        <input type="text" ng-model="filterObj.letters" ng-change="filterByLetters()">
+        <button class="filter" ng-click="filterByDates()">Order</button>
+    </div>
+    <table>
+        <tr>
+            <td>
+                <ul>
+                    <li ng-repeat="task in tasks" ng-if="isTaskVisible(task,'done')">
```

Comment:
Is there any reason you split list in 2 parts? 
It traverses array twice, please optimize it.